client/src/js/agreement/AgreementsTable.jsx

```diff
@@ -51,9 +51,11 @@ function AgreementsTable (props) {
     }
   ]
 
-  if (props.type === AgreementType.INDIVIDUAL) {
-    // remove the org name if we're printing individual CLAs
-    cols.shift()
+  // If there are extra columns, append them to the table, before the actions column
```

Comment:
this seems a little convoluted, we can probably change the logic so that:
```
let base_cols = [...]
const action_cols = [...]

if (props.extra_cols && props.extra_cols.length > 0) {
  base_cols = [...base_cols, ...props.extra_cols] 
}

const cols = [...base_cols, ...action_cols]
```

In this way we don't have to shift arrays around